src/Modules/Workspaces/Commands.Workspaces/RestorePowerBIWorkspace.cs

```diff
@@ -60,7 +60,7 @@ protected override void BeginProcessing()
 
             if (this.Scope == PowerBIUserScope.Organization)
             {
-                this.Logger.WriteWarning($"Only preview workspaces are supported when -{nameof(this.Scope)} {nameof(PowerBIUserScope.Organization)} is specified");
+                this.Logger.WriteWarning($"Only workspaces created in the new workspace experience are supported when -{nameof(this.Scope)} {nameof(PowerBIUserScope.Organization)} is specified");
```

Comment:
You might want to spot check that the other warning messages about v2 workspaces are consistent with this wording.